src/Compilers/CSharp/Portable/Symbols/Source/FieldSymbolWithAttributesAndModifiers.cs

```diff
@@ -395,8 +395,6 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                     DeclaringCompilation.SynthesizeTupleNamesAttribute(type.TypeSymbol));
             }
 
-            // PROTOTYPE(NullableReferenceTypes): type.ReportAnnotatedUnconstrainedTypeParameterIfAny()
```

Comment:
:memo: This is already checked elsewhere (`SourceFieldSymbol. AfterAddingTypeMembersChecks` calls `this.Type.ReportAnnotatedUnconstrainedTypeParameterIfAny(location, diagnostics);`)